### PR TITLE
Adjustment of STDCM interface (Part 1)

### PR DESCRIPTION
**Pull request by kmer2016**
There are some differences between [mockup](https://www.sketch.com/s/15e0909c-f17d-4200-a770-b9fc1154fa7e/p/CA8C9E9A-B795-4C87-9CF5-508DD0B3634A/canvas) and the STDCM interface implementation. This PR fix three of them
- [x] #8730 (first commit)
- [x] #8731 (second commit)
- [x] #8757 (second commit)

### CODE REVIEW

File: front/src/styles/scss/_uiCoreIntegration.scss [hunk 1]
```diff
@@ -68,3 +68,61 @@ img {
     }
   }
 }
+
+// __________________ Select  __________________
```
Comment (SharglutDev) [inline front/src/styles/scss/_uiCoreIntegration.scss:72]: Maybe we can add a todo here to remove this when the bootstrap/tailwind conflict will be resolved ?

File: front/src/styles/scss/_uiCoreIntegration.scss
Comment (SharglutDev) [inline front/src/styles/scss/_uiCoreIntegration.scss:1]: The position of the items list when opening the select seems to be wrong for me :/ (both on chrome and firefox)

mockup : 
Capture d’écran 2024-09-10 à 09 49 25

chrome :
Capture d’écran 2024-09-10 à 09 47 52

firefox : 
Capture d’écran 2024-09-10 à 09 44 41

**Review by SharglutDev — COMMENTED** [review]
Thank you for the PR ! Consist/itinerary and blue>black label fixes are good for me but still have some issues with the select one.

File: front/src/styles/scss/_uiCoreIntegration.scss
Comment (SharglutDev) [inline front/src/styles/scss/_uiCoreIntegration.scss:1]: Oh and for the arrow fix, when opening the select and closing it by clicking outside or on an item, the arrow seems to be the wrong one : 
Capture d’écran 2024-09-10 à 09 53 19

Comment (SharglutDev) [reply front/src/styles/scss/_uiCoreIntegration.scss:1]: Ok turns out it comes from a macOS internal config. This behavior is normal on a linux/windows device.

**Comment by Caracol3** [comment]
image

image


If the width should be fixed, are we sure to want to keep the icon ? When french langage is on, it prevents to put 'code de composition on one line'. Or maybe we should put it under the icon ?

If we keep it like that, i think it would be good to at least add a padding between the icon and the text. For me, the two are too close.

EDIT: in the sketch its called 'code compo.' , this solves the problem.

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested

**Comment by Caracol3** [comment]
In consist component, like 'tracion engine', 'speed limits' and 'composition code' should be bold.

**Comment by kmer2016** [comment]
Thank you for your review, @Caracol3. I will address your feedback in a separate [PR](https://github.com/OpenRailAssociation/osrd/issues/8820) where I will directly implement the Code Compo. component.

**Review by Caracol3 — APPROVED** [review]
lgtm

**Pull request merged**